var/spack/repos/builtin/packages/nco/package.py

```diff
@@ -38,23 +38,12 @@ class Nco(AutotoolsPackage):
 
     # See "Compilation Requirements" at:
     # http://nco.sourceforge.net/#bld
-    variant('mpi', default=True)
-
     depends_on('netcdf')
     depends_on('antlr@2.7.7+cxx')  # required for ncap2
     depends_on('gsl')              # desirable for ncap2
     depends_on('udunits2')         # allows dimensional unit transformations
```

Comment:
It's a mess due to inconsistent branding of UDUNITS vs UDUNITS-2 coming from upstream.  Feel free to suggest changing it, but ONLY after you've researched the issue.  See here, for an explanation, plus a living example of the inconsistent branding:

http://www.unidata.ucar.edu/software/udunits/udunits-2.1.24/udunits2.html